service/service.go

```diff
@@ -73,21 +75,32 @@ func (m *DefaultManager) Find(caps session.Caps, requestId uint64) (Starter, boo
 	if !ok {
 		return nil, false
 	}
-	switch service.Image.(type) {
-	case string:
-		if m.Client == nil {
-			return nil, false
-		}
-		log.Printf("[%d] [USING_DOCKER] [%s] [%s]", requestId, browserName, version)
-		return &Docker{
+	if m.Environment.MesosMasterUrl != "" || m.Environment.Zookeeper != "" {
```

Comment:
please move mesos section to switch under string case and there choose to use docker or mesos